docs/admins.rst

```diff
@@ -113,6 +119,53 @@ the x86_64 platform will publish v1 images as well as v2 images. Other
 platforms on instance1, and all platforms on instance2, will only
 publish v2 images.
 
+Autorebuilds and OSBS updates
```

Comment:
I think most of the text in this section can be removed. It was helpful in helping providing background to implement support for reactor config map.

:bomb: it?

The bit about using ImageStreams instead of images directly is still relevant.

Reply:
Eh, alright. But...